build.gradle

```diff
@@ -32,6 +32,12 @@ repositories {
     maven {
         url uri('https://nexus.fearnixx.de/repository/maven-public')
     }
+    maven {
+        url uri('https://jitpack.io')
+    }
+    maven {
+        url uri('http://tomp2p.net/dev/mvn/')
+    }
```

Comment:
We really need a better solution for this. Maybe we should set up a mirror for those repositories on our Nexus and include them in the ``maven-public`` group. That would obviously be a thing I would have to do.  
Opinions? @genius42

Reply:
:+1: